Emoji filesize limit

pointydev opened the pull request:
Added JSON error related to strings added in [canary build 9856](https://github.com/DJScias/Discord-Datamining/commit/fb119e36a95349a3506a79a73e642bc07d20d7bf), also seems to be live on API in an unfinished state, so I've PR'd with placeholders.

Thanks,
Pointy

pointydev committed:
Emoji filesize limit

Added JSON error related to strings added in canary build 9856 (WIP)

pointydev committed:
Update emoji filesize to 256kb

Per @msciotti's first comment on PR #493

night commented on docs/topics/Opcodes_and_Status_Codes.md:
```diff
@@ -149,6 +149,7 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 50036 | An invite was accepted to a guild the application's bot is not in |
 | 50041 | Invalid API version |
 | 90001 | Reaction blocked |
+| XXXXX | Emojis cannot be larger than 256kb |
```
this is a form error, not a normal api error, which unfortunately will not return an error code .. until we deploy new form error responses you won't be able to document it

pointydev committed:
Revert "Update emoji filesize to 256kb"

This reverts commit d41186864bfb85448a02faed56cf73324ba9935a.

pointydev committed:
Revert "Emoji filesize limit"

This reverts commit 232464b9a58bdb8ad15cbf71d041d31ed275d34c.

pointydev committed:
Emoji filesize limit note

See
https://github.com/discordapp/discord-api-docs/pull/493#issuecomment-361369261

pointydev committed:
Remove one notice from top of Emoji.md

See
https://github.com/discordapp/discord-api-docs/pull/493#issuecomment-362373015
(silly Mason)

msciotti committed:
Wording nits
